pages/docs/concepts/asyncapi-document/adding-operations.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Adding Operations
+weight: 60
+---
+
+In a messaging system, the term "operations" refers to the various methods by which messages are exchanged between participants or components. 
+
+## Features
+
+- Operations describe the behaviors and capabilities of the messaging channels described in the AsyncAPI document.
+
+- In a messaging channel, an operation represents a particular action or interaction that can be performed. 
+
+- The purpose of these operations is to provide a standardized means for describing the process of sending, receiving from, requesting, or replying to messages within the messaging system.
+
+## Defining Operations
+
+Operations can be defined as an independent object in the AsyncAPI document. Operations have the following components for it's definition. More information about each field names that are used to define operations can be found [here](https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#operationObject). 
```

Comment:
> Operations have the following components for it's definition

sorry if I judge too quickly and draw false conclusions, but looking at this round of review, you did not really read what you wrote before submitting for another review round. Please make sure that what you write is accurate and ready for review (relating also to other comments where I noticed you just copy over stuff without making sure what you do)

Reply:
Sorry for the previous miscommunication. I have made the required changes and also updated the contents with some details about the field names in adding operations. I have ensured all changes are pushed correctly this time.